### PR TITLE
More QA requests

### PR DESCRIPTION
**Pull request by bruugey**
- Updates design docs to leverage 16px base font size
- Use our Table component instead of html table elements
- Removes old next config
- Makes knobs accessible

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -1,69 +0,0 @@
-const nextConfig = {
-  reactStrictMode: true,
```
Comment (stephl3) [inline next.config.js:2]: do we want to keep this on in the new config?

File: src/components/content-stack/AnnotatedImageBlock/StyledListItem.tsx [hunk 2]
```diff
@@ -32,6 +32,11 @@ export const StyledListItem = ({ children }: { children: React.ReactNode }) => {
           box-shadow: 1px 1px 0px ${color[theme].border.primary.default};
           margin-top: 3px;
         }
+
+        & > p {
+          font-size: 16px;
+          line-height: 28px;
+        }
```
Comment (stephl3) [inline src/components/content-stack/AnnotatedImageBlock/StyledListItem.tsx:39]: isn't this redundant if you apply these styles to the typography components in the `componentMap` file?

File: src/components/content-stack/ExampleCardBlock/ExampleCardBlock.tsx [hunk 3]
```diff
@@ -55,13 +55,15 @@ export const ExampleCardBlock = ({
...
           <Body
+            baseFontSize={16}
             className={css`
               color: ${TextColors(theme)[entry.variant]};
             `}
           >
             <b>{VariantHeaders[entry.variant]}</b>
           </Body>
           <Body
+            baseFontSize={16}
```
Comment (stephl3) [inline src/components/content-stack/ExampleCardBlock/ExampleCardBlock.tsx:66]: `typeScales.body2.fontSize` token?

File: src/components/content-stack/componentMap.tsx [hunk 4]
```diff
@@ -1,6 +1,14 @@
...
 import { palette } from '@leafygreen-ui/palette';
+import {
+  Table,
+  TableHead,
+  TableBody,
+  Row,
+  Cell,
+  HeaderCell,
+} from '@leafygreen-ui/table';
 import { spacing } from '@leafygreen-ui/tokens';
```
Comment (stephl3) [inline src/components/content-stack/componentMap.tsx:12]: can we use typeScales tokens for the font-size and line-height attributes in this file?

File: src/components/content-stack/componentMap.tsx [hunk 5]
```diff
@@ -100,6 +108,7 @@ export const nodeTypeToElementMap: {
   ),
   [CSNodeType.PARAGRAPH]: (node, props) => (
     <Body
+      baseFontSize={16}
```
Comment (stephl3) [inline src/components/content-stack/componentMap.tsx:111]: maybe can add the line-height in the `className` here?

File: src/components/global/SideNavigation/SideNavigation.tsx [hunk 6]
```diff
@@ -249,6 +249,9 @@ export function SideNavigation() {
             key="logo"
             height={24}
             color={darkMode ? SupportedColors.White : SupportedColors.Black}
+            className={css`
+              z-index: 0;
+            `}
```
Comment (stephl3) [inline src/components/global/SideNavigation/SideNavigation.tsx:254]: is this doing anything?

File: src/components/live-example/KnobRow.tsx [hunk 7]
```diff
@@ -40,30 +40,31 @@ interface KnobRowProps extends HTMLElementProps<"div"> {
 }
 
 export const KnobRow = ({ knob, knobValue, setKnobValue }: KnobRowProps) => {
-  const { control = "text", name, options = [] } = knob;
+  const { control = 'text', name, options = [] } = knob;
   const { theme } = useDarkMode();
+  const knobLabel = kebabCase(`knob-${name}`);
 
-  if (control === "none") return null;
+  if (control === 'none') return null;
```
Comment (stephl3) [inline src/components/live-example/KnobRow.tsx:47]: unrelated to your change but would prefer these being enums
e.g. `Knob.Text` and `Knob.None`

**Review by stephl3 — COMMENTED** [review]
The logo hover issue is still showing up for me on local. To repro, on desktop breakpoints, mouse to sidenav and hover over the top half of the logo. It is covered by the top nav

To fix, you can:
1. remove the `width: 100%;` on the top nav, or
2. add `z-index: 1;` to the `<nav/>` tag in the `SideNav` component

File: src/components/content-stack/AnnotatedImageBlock/StyledListItem.tsx (hunk 2 above)
Comment (bruugey) [reply src/components/content-stack/AnnotatedImageBlock/StyledListItem.tsx:39]: its not unfortunately a Body tag gets auto inserted and it comes from the provider value set at the root

File: src/components/content-stack/ExampleCardBlock/ExampleCardBlock.tsx (hunk 3 above)
Comment (bruugey) [reply src/components/content-stack/ExampleCardBlock/ExampleCardBlock.tsx:66]: we don't need that here

File: src/components/global/SideNavigation/SideNavigation.tsx (hunk 6 above)
Comment (bruugey) [reply src/components/global/SideNavigation/SideNavigation.tsx:254]: allows you to hit the svg

File: src/components/live-example/KnobRow.tsx (hunk 7 above)
Comment (bruugey) [reply src/components/live-example/KnobRow.tsx:47]: half of the values are JS keywords - which I imagine is why this wasn't done in the first place 😓

File: src/components/content-stack/componentMap.tsx (hunk 4 above)
Comment (bruugey) [reply src/components/content-stack/componentMap.tsx]: updated!

File: next.config.js (hunk 1 above)
Comment (bruugey) [reply next.config.js:2]: i believe its in by default

**Review by stephl3 — APPROVED** [review]

**Pull request merged**